common/notification/index.ts

```diff
@@ -239,3 +247,20 @@ async function deliverNotification(concreteNotification: ConcreteNotification, n
     }
 }
 
+export async function createAttachments(files: Express.Multer.File[], uploader: Student | Pupil): Promise<AttachmentGroup> {
```

Comment:
Nice! What happens if no files are passed in (empty array)? Wouldit make sense to return `null` in that case? As far as I can see this currently generates a group ID with no corresponding files in the attachments table.

Reply:
Indeed, it would make more sense to just return `null`, especially considered that all the other notification methods take in the attachments as an optional parameter 👍